ui-kit/src/main/res/color/button_lime_to_mountain_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
i would rather to name this file with prefix `selector`
it's not a class to call it as button